### PR TITLE
Make TLSVerificationHandler internal

### PR DESCRIPTION
**Pull request by glbrntt**
See #657

### CODE REVIEW

File: Sources/GRPC/TLSVerificationHandler.swift [hunk 1]
```diff
@@ -37,8 +37,8 @@ public enum GRPCApplicationProtocolIdentifier: String, CaseIterable {
 /// Users of this handler should rely on the `verification` future held by this instance.
 ///
 /// On fulfillment of the promise this handler is removed from the channel pipeline.
-public class TLSVerificationHandler: ChannelInboundHandler, RemovableChannelHandler {
-  public typealias InboundIn = Any
+internal class TLSVerificationHandler: ChannelInboundHandler, RemovableChannelHandler {
+  internal typealias InboundIn = Any
```
Comment (MrMage) [inline Sources/GRPC/TLSVerificationHandler.swift]: Given that the entire class is `internal`, shall we drop that modifier on the internal fields and methods?

Comment (glbrntt) [reply Sources/GRPC/TLSVerificationHandler.swift]: I prefer the explicit `internal` but fine changing if you prefer?

Comment (MrMage) [reply Sources/GRPC/TLSVerificationHandler.swift]: Not feeling too strongly, but it does feel quite redundant to me if the same visibility is already specified on the enclosing scope. I think Swift even warns for these redundant visibilities in some cases.

Comment (glbrntt) [reply Sources/GRPC/TLSVerificationHandler.swift]: Haven't seen the warnings but let's remove them anyway.

**Pull request merged**